web-client/public/js/setup-handlers.js

```diff
@@ -110,38 +108,38 @@ export const setupHandlers = grnState => {
                     }
                 }
             };
-            visit(svg);
+            visit(node);
             return tree;
         };
 
         const explicitlySetStyle = element => {
             const cSSStyleDeclarationComputed = window.getComputedStyle(element);
```

Comment:
I realize this was pre-existing code, but since you’re already changing this, let’s fix the capitalization issue: the `CSS` here should actually be all lowercase, if we are to follow the convention:
```suggestion
            const cssStyleDeclarationComputed = window.getComputedStyle(element);
```